graylog2-server/src/main/java/org/graylog/scheduler/eventbus/JobSchedulerEventBus.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog.scheduler.eventbus;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import com.google.common.eventbus.EventBus;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+/**
+ * Job scheduler specific event bus instance. This is a <em>synchronous</em> event bus.
+ * Subscribers must ensure that the callback method is fast or put expensive work into an executor or queue.
+ */
+public class JobSchedulerEventBus extends EventBus {
+    private static final String METRICS_PREFIX = "job-scheduler-eventbus";
```

Comment:
This won't be shown on the metrics page, because the default filter needs everything to start with `org.`

Why not use:

```
@@ -28,16 +28,14 @@ import static com.codahale.metrics.MetricRegistry.name;
  * Subscribers must ensure that the callback method is fast or put expensive work into an executor or queue.
  */
 public class JobSchedulerEventBus extends EventBus {
-    private static final String METRICS_PREFIX = "job-scheduler-eventbus";
-
     private final Counter registrationsCount;
     private final Timer postTimer;
 
     @SuppressWarnings("WeakerAccess")
     public JobSchedulerEventBus(String name, MetricRegistry metricRegistry) {
         super(name);
-        this.registrationsCount = metricRegistry.counter(name(METRICS_PREFIX, name, "registrations"));
-        this.postTimer = metricRegistry.timer(name(METRICS_PREFIX, name, "posts"));
+        this.registrationsCount = metricRegistry.counter(name(this.getClass(), name, "registrations"));
+        this.postTimer = metricRegistry.timer(name(this.getClass(), name, "posts"));
     }
 
     @Override
```

Reply:
Good point! :+1: Will fix it.